client/verta/verta/_cli/registry/update.py

```diff
@@ -103,4 +92,17 @@ def update_model_version(model_name, version_name, label, model, custom_module,
         model_version.log_environment(Python(requirements=reqs))
 
     if description:
-        registered_model.set_description(description)
+        model_version.set_description(description)
+
+    add_attributes(model_version, attribute, overwrite)
+
+
+def add_attribute(model_version, key, value):
+    model_version.add_attribute(key, json.loads(value))
+
+
+def add_attributes(model_version, attribute, overwrite):
+    attributes = MultipleArguments(attribute, "attribute")
+    attributes.for_each(
+        lambda key, value: add_attribute(model_version, key, value),
+        lambda: model_version._get_attribute_keys(), overwrite)
```

Comment:
It is confusing for me that there are util functions called `add_attribute()`, when `ModelVersion` _also_ has `add_attribute()` methods. And these don't seem to be doing a lot. Could we remove these functions and just put this code in `update_model_version`?

Reply:
I agree about add_attribute. I just created it because I thought it will be doing more parsing like in these cases: '3.6'. Because json doesn't support single quotes. But it should be alright with the plain json like: "3.6", right? So I inlined it now.

But add_attributes is needed because it is used to test the feature without calling any additional things from model version update.